contributor-guides/	translating-astro-docs.md

```diff
@@ -0,0 +1,359 @@
+
+### How can I help translate/review one of the supported languages?
+
+Translations all live in this GitHub repository. You can add and update them by creating a pull request or reviewing pending translations in the "Pull Requests" tab. Read on to find out more!
+
+#### Minimum quality standards
+
+We are only able to maintain multiple translations of the docs thanks to our amazing team of volunteers, who are prepared to spot typos and fix the occasional grammatical error. Translations requiring significant editing cannot be accepted and therefore, we may close PRs that don't meet our minimum quality standards.
+
+We do expect translators to be Astro users, so that they are familiar with Astro terms and concepts. Translations should be faithful representations of the English text, at a native speaker's grammar level, without changes to the meaning and structure of a page. Changes to content are only made in the English documentation, and translations that attempt to "improve" the documentation will not be accepted. (If you believe there is a problem with the English documentation, then please create a GitHub Issue so it can be addressed in the English docs.) This ensures translation reviewers will only have to make small (if any) suggestions to a translation before merging, making our current workflow sustainable.
+
+### How can I find out what needs to be reviewed/translated?
+
+To find translations pending review, you can filter through this repo's [Pull Requests with the i18n label](https://github.com/withastro/docs/pulls?q=is%3Apr+is%3Aopen+label%3Ai18n).
+
+See our automated [Translation Status Overview](https://i18n.docs.astro.build/) for a quick list of which pages are missing a translation or need updating to match a change to the English version.
+
+> **Warning**
+> Please do not translate any pages without first checking their status in our [Overview Issue](https://github.com/withastro/docs/issues/438)! If a page is not listed here as needing a translation or an update, we can not accept your PR.
+ 
+Not every page is marked as "ready to translate." So, even if you find a page that is not yet translated on the Docs site, you must first confirm that it is on the list of available pages to translate. Do not translate documents that are missing:
+
+- A "Translate this page" button in the Docs site.
+- The `i18nReady: true` frontmatter property in its MDX file.
+
+You can read more about how pages are marked "ready for (initial) translating" and "needs updating" in [CONTRIBUTING.md](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md).
+
+## Languages for translation
+
+Currently we are aiming to translate the Astro documentation into the following languages:
+
+- Brazilian Portuguese
+- Chinese (Simplified)
+- English
+- French
+- German
+- Japanese
+- Spanish
+- Korean
+- Polish
```

Comment:
I think it would be great to follow alphabetical order, what do you think?
```suggestion
- Brazilian Portuguese
- Chinese (Simplified)
- English
- French
- German
- Japanese
- Korean
- Polish
- Spanish
```

Reply:
That does make more sense @dreyfus92! :)